cypress/support/commands.js

```diff
@@ -23,3 +23,4 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import 'cypress-axe'
```

Comment:
Since `cypress-axe` is imported to `support/index.js` (which also imports this file) does it need to be here at all?

Reply:
you're right -- seems to work fine without. pushing up the change now!